locale/en/user.po

```diff
@@ -750,13 +751,13 @@ msgstr ""
 "\tLearn more in <a href=\"https://info.orcid.org/"
 "whats-so-special-about-signing-in/\">What’s so special about signing "
 "in.</a><br>\n"
-"This journal will collect and display authenticated authors’ and coauthors’ "
-"iDs on the OJS profile and article page."
+"This site will collect and display authenticated authors’ and coauthors’ "
+"iDs on the user profile and publication landing page."
```

Comment:
We have to be a little cautious with "publication" as preprints are posted not published. Maybe "and where authorship is listed"?

Reply:
Thanks! I was looking for something like that!